geometry_msgs/Point cp1  # contact point 1
geometry_msgs/Point cp2  # contact point 2
geometry_msgs/Quaternion orientation
float32 width